### PR TITLE
442 ui layout redux state

### LINKED ISSUE #442: show filter on page (redux state)
Move logic to show Filter into redux store, and hide it completely on the detail page

- [ ] move the logic for including the left side filter element into the redux store
- [ ] the collection detail should update it to not display at all
- [ ] include if applied filter bubbles should be shown, or refactor the detail view so that they don't appear above the selected collection (possibly just change the nesting of routes in index.jsx)

### PR DESCRIPTION
**Pull request by zebdelk**
resolves #442

### CODE REVIEW

File: client/src/reducers/ui/layout.js [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
 })
 
 export const layout = (state = initialState, action) => {
   switch (action.type) {
-    case TOGGLE_LEFT:
-      const previousShowLeft = state.showLeft
-      return Immutable.set(state, 'showLeft', !previousShowLeft)
+    case LOCATION_CHANGE:
+      const path = action.payload.pathname
+      const is508 =
```
Comment (forgo) [inline client/src/reducers/ui/layout.js]: Since this logic is in at least two places, should it be a utility function somewhere?

Comment (zebdelk) [reply client/src/reducers/ui/layout.js]: I completely agree :)
Take a look at my branch for 445 (which is the WIP for the granule list stuff)

Comment (zebdelk) [reply client/src/reducers/ui/layout.js]: I cherry picked these from the other branch.

**Comment by ajakz** [comment]
image

**Pull request merged**